src/cljs/athens/views/buttons.cljs

```diff
@@ -23,41 +38,49 @@
    :border           "none"
    :display          "inline-flex"
    :align-items      "center"
-   :color            "rgba(50, 47, 56, 1)"
+   :color            (color :body-text-color)
    :background-color "transparent"
-   :transition       "all 0.05s ease"
-   ::stylefy/mode [[:hover {:background-color "#EFEDEB"}]
-                   [:active {:color "rgba(0, 117, 225)"
-                             :background-color "rgba(0, 117, 225, 0.1)"}]
-                   [:disabled {:color "rgba(0, 0, 0, 0.3)"
-                               :background-color "#EFEDEB"
-                               :cursor "default"}]]
-   ::stylefy/manual [[:svg {:margin-block-start "-0.0835em"
-                            :margin-block-end "-0.0835em"}
-                      [(selectors/& (selectors/not (selectors/last-child))) {:margin-inline-end "0.251em"}]
-                      [(selectors/& (selectors/not (selectors/first-child))) {:margin-inline-start "0.251em"}]
-                      [(selectors/& ((selectors/first-child (selectors/last-child)))) {:margin-inline-start "-4px"
-                                                                                       :margin-inline-end "-4px"}]]
+   :transition       "all 0.075s ease"
+   ::stylefy/manual [[:&:hover {:background (darken (color :panel-color :opacity-low) 10)}]
+                     [:&:active
+                      :&:hover:active
+                      :&.active {:color (color :body-text-color)
+                                 :background-color  (darken (color :panel-color :opacity-med) 10)}]
+                     [:&:disabled {:color (color :body-text-color 0.3)
```

Comment:
Do we want to set `cursor: none` if disabled?

Reply:
I don't usually see special cursors on disabled buttons, usually it’s just the default arrow, rather than the pointing finger.